src/Apps/ConfigAppManager.php

```diff
@@ -30,11 +30,9 @@ public function __construct()
      */
     public function all(): array
     {
-        return $this->apps
-            ->map(function (array $appAttributes) {
-                return $this->convertIntoApp($appAttributes);
-            })
-            ->toArray();
+        return $this->apps->map(function (array $appAttributes) {
+            return $this->convertIntoApp($appAttributes);
+        })->toArray();
```

Comment:
:thinking: I think this is definitely worse. Should configure the style tooling you used to support the previous way I think.

Reply:
Wanted to flip from this styling with a lot of safety for future diff lines to Laravel's way of doing things more in-line-ish. I think I'll be reverting them.

Reply:
Yeah the problem is that functional programming with chained `->then()` and whatnot is inherently incompatible from a style perspective with stock Laravel IMO. It's more JS-like in a sense.